### ISSUE TITLE
Can't view contents of added files in commits

### ISSUE THREAD

**Issue by akaihola** [EddieRingle/hubroid#71]
When I click on added files in a commit, I get the list of added files, but clicking on them does not open the contents of the file. 

Viewing diffs for changed files does work well. 

_Sent via Hubroid_

**Comment by EddieRingle** [comment]
Yes, I'll fix this when I finish converting to using v3 of the API. :)
